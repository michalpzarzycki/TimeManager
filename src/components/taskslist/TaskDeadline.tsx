import React from 'react';
import {formatDistance} from 'date-fns'
import styles from './TaskDeadline.module.css'

export default function TaskDeadline({task}: any) {
    return(
        <div className={styles.deadline}>
            {task.deadline && formatDistance(new Date(task.deadline), Date.now(),{ addSuffix: true })}
        </div>
    )
}