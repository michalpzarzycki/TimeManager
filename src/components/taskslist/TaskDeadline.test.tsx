import React from 'react'
import {shallow} from 'enzyme'
import TaskDeadline from './TaskDeadline'

test('Expext to render TaskDeadline component', () => {
    expect(shallow(<TaskDeadline task={{}}/>)).toMatchSnapshot()
})
