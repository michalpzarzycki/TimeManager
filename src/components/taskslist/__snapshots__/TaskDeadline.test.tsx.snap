// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expext to render TaskDeadline component 1`] = `ShallowWrapper {}`;
